benchmark/src/androidTest/java/com/wire/benchmark/BaselineGenerator.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.benchmark
+
+import androidx.benchmark.macro.junit4.BaselineProfileRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BaselineGenerator {
+
+    @get:Rule
+    val baselineProfileRule = BaselineProfileRule()
+
+    @Test
+    fun startup() = baselineProfileRule.collect(
+        packageName = PACKAGE_NAME
+    ) {
+        pressHome()
+        startActivityAndWait()
+    }
+
+    companion object {
+        private const val PACKAGE_NAME = "com.wire.android.internal"
+    }
+}
```

Comment:
Suggestion: can we @ignore this test, as it currently does not provide feedback when running on automation?

Reply:
it's already ignored, we don't run this on our CI

Reply:
It's not really ignore, it's just that we're explicitly running `:app:xxxTest` tasks.

I think it's ok to leave it like this, we just gotta be careful to not start running all tests by accident on CI.